tests/app/config/config.yml

```diff
@@ -36,7 +36,6 @@ pomm:
     configuration:
         my_db1:
             dsn: "pgsql://%database_user%:%database_password%@%database_host%:%database_port%/%database_name%"
-            class:session_builder: '\PommProject\ModelManager\SessionBuilder'
```

Comment:
Can you let `my_db1` untouched to prevent regretion? Add a second connection if you need other parameters.